app/pages/cif/project-revision/[projectRevision]/index.tsx

```diff
@@ -163,7 +182,7 @@ export function ProjectRevision({
           >
             Submit
           </Button>
-          <Button size="medium" variant="secondary">
+          <Button size="medium" variant="secondary" onClick={discardRevision}>
```

Comment:
These buttons should be disabled if either the submit or discard mutations are in flight. 

I wouldn't disable them if `updatingFormChange` is true though, as it would lead to some distracting flickering, with the buttons changing state as you type. There is no risk to leaving them enabled in that case as the order of mutations would be preserved.